updater/test/Main.purs

```diff
@@ -0,0 +1,11 @@
+module Test.Main where
```

Comment:
Could remove

Reply:
I'd like to keep it in as tests should be added at some point.